cpp/src/slice2java/JavaUtil.h

```diff
@@ -65,6 +65,12 @@ namespace Slice
 
         void close();
 
+        //
+        // Check a symbol against any of the Java keywords. If a
```

Comment:
I fixed it.
But this function was just changed from `protected` to `private`, and I think it'd be better to open a dedicated PR for fixing all the comments instead of just piecemeal fixing the ones that we come across. Otherwise we're going to have a mix of comments that never gets fixed.